frontend/Ingredishare/app/src/main/java/com/example/andriod/ingredishare/MainActivity.java

```diff
@@ -75,35 +65,6 @@
     private GlobalRequestQueue mReqQueue;
     private RequestQueue requestQueue;
 
-
-    /*
-    Create a getRequestQueue() method to return the instance of
-    RequestQueue.This kind of implementation ensures that
-    the variable is instatiated only once and the same
-    instance is used throughout the application
-    */
-    public RequestQueue getRequestQueue() {
-        if (requestQueue == null)
-            requestQueue = Volley.newRequestQueue(getApplicationContext());
-        return requestQueue;
-    }
-    /*
-    public method to add the Request to the the single
-    instance of RequestQueue created above.Setting a tag to every
-    request helps in grouping them. Tags act as identifier
-    for requests and can be used while cancelling them
-    */
-    public void addToRequestQueue(Request request, String tag) {
-        request.setTag(tag);
-        getRequestQueue().add(request);
-    }
-    /*
-    Cancel all the requests matching with the given tag
-    */
-    public void cancelAllRequests(String tag) {
-        getRequestQueue().cancelAll(tag);
-    }
-
```

Comment:
these methods in `MainActivity.java` aren't being used anymore, so I took them out.